packages/jest-cli/src/plugins/update_snapshots_interactive.js

```diff
@@ -25,7 +25,6 @@ class UpdateSnapshotInteractivePlugin extends BaseWatchPlugin {
     super(options);
     this._failedSnapshotTestAssertions = [];
```

Comment:
this as well?

Reply:
You can move all of these to properties, even `new SnapshotInteractiveMode(this._stdout);`.